homeassistant/components/screenlogic/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Pentair ScreenLogic",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/screenlogic",
-  "requirements": ["screenlogicpy==0.6.4"],
```

Comment:
Change iot_class to local_push.

Reply:
Didn't know if it qualified not being _completely_ push.

Reply:
We use the best method that the integration supports so all good to use push here